projects/fsastorefrontlib/src/recipes/storefront.module.ts

```diff
@@ -101,6 +102,7 @@ export class FSStorefrontModule {
         provideConfig(config),
         provideDefaultConfigFactory(defaultFSGlobalMessageConfigFactory),
         provideDefaultConfigFactory(fsDefaultDateFormatConfigFactory),
+        ...defaultCmsContentProviders,
```

Comment:
Try to build application in production mode and test this feature. I remember we had issues in production when using spread operator.